pageserver/src/pgdatadir_mapping.rs

```diff
@@ -90,6 +92,24 @@ pub enum LsnForTimestamp {
     NoData(Lsn),
 }
 
+#[derive(Debug, Clone, Copy, Default)]
+pub struct LsnRange {
```

Comment:
nit: a comment would be nice:
* explain difference between effective and request lsn
* explain that primary compute always uses `request_lsn == MAX`
* hint that this is how the compute thinks about get page requests

Reply:
Done